src/components/DocumentationTopic/DocumentationNav/HierarchyItem.vue

```diff
@@ -103,11 +101,11 @@ export default {
   }
 }
 
-$-max-uncollapsed-breadcrumbs: 3;
+$-max-uncollapsed-breadcrumbs: 5;
```

Comment:
What's the difference between this and the `MaxVisibleItems` variable in `Hierarchy.vue`?

Reply:
This one counts all items, except the root - `technology / link_1 / link_2 / link_3 / current_page`

Reply:
![image](https://user-images.githubusercontent.com/9863944/154438321-223abc0c-bc48-47a2-a610-e7546860862b.png)